packages/terra-application/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "terra-application",
```

Comment:
Is this going to replace this beta [terra-application](https://www.npmjs.com/package/terra-application)?

Reply:
Yeah, that's the plan.